exasol/analytics/query_handler/query/select.py

```diff
@@ -125,8 +130,13 @@ def __init__(
         self._audit = audit
 
     @property
-    def db_object_name(self) -> DBObjectName:
-        return self.audit_fields[DB_OBJECT_NAME_TAG]
+    def db_object_name(self) -> str:
+        return self.db_object_ref.name
+
+    @property
+    def db_object_schema(self) -> Optional[str]:
+        schema = self.db_object_ref.schema_name
+        return schema.name if schema else None
```

Comment:
what is the purpose of these properties

Reply:
I don't know. 
Most properties had been added in PR #256.
I am happy to remove these properties. 
Should I?

Reply:
see next push 
* not supporting string values anymore but only objects `DbObjectType`, `DBObjectName`, `DbSpanType`
* using private attributes and public (read-only) properties 
* not updating `audit_fields`